public/index.ts

```diff
@@ -632,7 +633,8 @@ function showChannelUI(volume: Volume) {
       // this doesn't give good results currently but is an example of a per-channel button callback
       autoIJ: (function (j) {
         return function () {
-          const lut = volume.getHistogram(j).lutGenerator_auto2();
+          const [hmin, hmax] = volume.getHistogram(j).findAutoIJBins();
+          const lut = new Lut().createFromMinMax(hmin, hmax);
```

Comment:
@frasercl This gets slightly more verbose because it was decoupled.  Many of the Luts can be created in these two steps: Get info from histogram, and then pass it to the minmax constructor.  But a couple of them actually do more complicated stuff and for those I just pass the whole Histogram in to the Lut construction function. (createLabelColors and createFromEqHistogram).  This change necessitates changes up at the w3cv level too.  Arguably one could just pass the whole histogram to all the Lut creation funcs instead.